dbt/seeds/pinval/pinval.vars_dict.csv

```diff
@@ -0,0 +1,118 @@
+Percent Households Nonfamily + Living Alone in Census Tract,acs5_percent_household_nonfamily_alone,"Percent of households that are non-family, alone (non-married), in the home's census tract (ACS 5-year estimate).",
+Percent Population with High School Degree in Census Tract,acs5_percent_education_high_school,Percent of people older than 25 who attained a high school degree in the home's census tract (ACS 5-year estimate).,
+Percent Population with Bachelor's Degree in Census Tract,acs5_percent_education_bachelor,Percent of people older than 25 who attained a bachelor's degree in the home's census tract (ACS 5-year estimate).,
+Percent Population with Graduate Degree in Census Tract,acs5_percent_education_graduate,Percent of people older than 25 who attained a graduate degree in the home's census tract (ACS 5-year estimate).,
+Percent Population Below Poverty Level in Census Tract,acs5_percent_income_below_poverty_level,Percent of people above the federal poverty line in the last 12 months in the home's census tract (ACS 5-year estimate).,
+Median Household Income in Past Year in Census Tract,acs5_median_income_household_past_year,Median income per household in the past 12 months in the home's census tract (ACS 5-year estimate).,
+Median Per Capita Income in Past Year in Census Tract,acs5_median_income_per_capita_past_year,Median income per capita in the past 12 months in the home's census tract (ACS 5-year estimate).,
+Percent Population Received SNAP in Past Year in Census Tract,acs5_percent_income_household_received_snap_past_year,Percent of households that received SNAP in the past 12 months in the home's census tract (ACS 5-year estimate).,
+Percent Population Unemployed in Census Tract,acs5_percent_employment_unemployed,Percent of people 16 years and older who are unemployed in the home's census tract (ACS 5-year estimate).,
+Median House Year Built in Census Tract,acs5_median_household_total_occupied_year_built,Median year built for all occupied households in the home's census tract (ACS 5-year estimate).,
+Median Gross Rent in Census Tract,acs5_median_household_renter_occupied_gross_rent,Median gross rent for only renter-occupied units in the home's census tract (ACS 5-year estimate).,
+Percent Owner-Occupied Households in Census Tract,acs5_percent_household_owner_occupied,Percent of households that are owner-occupied in the home's census tract (ACS 5-year estimate).,
+Year Built,char_yrblt,Year the home was constructed.,
+Central Air Conditioning,char_air,,
+Apartments,char_apts,Number of residential apartments in the building.,
+Attic Finish,char_attic_fnsh,,
+Attic Type,char_attic_type,,
+Bedrooms,char_beds,Number of bedrooms in the building.,
+Building Square Feet,char_bldg_sf,"Square footage of the building, as measured from the exterior.",
+Combined Bldg. Sq. Ft.,combined_bldg_sf,"Combined square footage of all buildings on the parcel.",

```

Comment:
Would you add more info about our approach here? Not sure if it is too much given we are already explaining in other areas